src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -443,6 +449,29 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                             .build()
                     )
                 }
+                is ContainerImageTreeNode -> {
+                    val issuesForImage = node.userObject as ContainerIssuesForImage
+                    val scrollPane = wrapWithScrollPane(
+                        BaseImageRemediationDetailPanel(project, issuesForImage)
+                    )
+                    descriptionPanel.add(scrollPane, BorderLayout.CENTER)
+
+                    val targetImage = project.service<KubernetesImageCache>()
+                        .getKubernetesWorkloadImages()
+                        .find { it.image == issuesForImage.imageName }
+                    val psiFile = targetImage?.psiFile
+                    val line = targetImage?.lineNumber?.let { it - 1 } // to 1-based count used in the editor
```

Comment:
Does IntelliJ use 0 for the first line in the document?

Reply:
it seems so